signup-service/src/test/java/io/mosip/signup/services/ChallengeManagerServiceTest.java

```diff
@@ -64,6 +65,18 @@ public void doGenerateChallenge_allValid_thenPass() throws SignUpException {
         Assert.assertEquals(challenge, "1111");
     }
 
+    @Test
+    public void doGenerateChallenge_withUnsupportedChallengeType_thenPass() throws SignUpException {
```

Comment:
this test case returns error code, pls change the method name to "doGenerateChallenge_withUnsupportedChallengeType_thenFail()"